content/scripts/build.js

```diff
@@ -616,7 +616,10 @@ class Builder {
     self.dumpAllURLs();
     self.summarizeResults(counts, t1 - t0);
     self.reportMacroRenderingFlaws();
-    if (self.reportMacroRenderingErrors()) {
+    if (
+      this.options.flaws === FLAWS_LEVELS.ERROR &&
+      this.self.reportMacroRenderingErrors()
+    ) {
```

Comment:
Hm, we should discuss this. The problem with this is that the macro-rendering errors stored in `this.macroRenderer.errors` are not considered flaws but hard errors (as they are currently considered in Kuma/Kumascript). I suppose we could consider them flaws? Also, just FYI, the way I have it setup currently, if you specify `--flaws=error`, the macro-renderer will throw hard errors when encountering flaws, so all of the flaws will become erros in `this.macroRenderer.errors`, and `this.macroRenderer.flaws` will be empty.

Reply:
Oh, I see you've commented on this already in your introduction, sorry I missed that until now! Perhaps we need to just meet and discuss this?